xbmc/interfaces/Builtins.cpp

```diff
@@ -300,7 +300,8 @@ void CBuiltins::GetHelp(std::string &help)
 bool CBuiltins::ActivateWindow(int iWindowID, const std::vector<std::string>& params /* = {} */, bool swappingWindows /* = false */)
 {
   // don't activate a window if there are active modal dialogs
-  if (g_windowManager.HasModalDialog() && !g_windowManager.GetWindow(iWindowID)->IsDialog())
+  if (g_windowManager.HasModalDialog() &&
+      !(g_windowManager.GetWindow(iWindowID) != NULL && g_windowManager.GetWindow(iWindowID)->IsDialog()))
```

Comment:
all those checks grab the gfx lock which is a no-go for interface functions. what is the benefit of having those, print a debug log and play some sound? if there is nothing else, those should go away.